website/templates/emails/node_request_institutional_access_request.html.mako

```diff
@@ -0,0 +1,35 @@
+<%inherit file="notify_base.mako" />
+
+<%def name="content()">
+    <tr>
+      <td style="border-collapse: collapse;">
+        <%!from website import settings%>
+        Hello ${recipient.fullname},
+        <p>
+            <a href="${sender.absolute_url}">${sender.fullname}</a> has requested access to <a href="${node.absolute_url}">${node.title}</a>.
+        </p>
+        % if comment:
+        <p>
+            ${comment}
+        </p>
+        % endif
+        <p>
+            To review the request, click <a href="${node.absolute_url}/contributors/">here</a> to allow or deny access and configure permissions.
+        </p>
+        <p>
+            This request is being sent to you because your project has the “Request Access” feature enabled.
+            This allows potential collaborators to request to be added to your project or to disable this feature, click
+            <a href="${node.absolute_url}/settings/">here</a>
```

Comment:
Is this true with institutional access requests? I don't think you can disable institutional admins requesting access, can you?

Reply:
Yeah I talked to Blaine about this and she indicated the users should ALWAYS be able to turn off request access, I'll double check though.

Reply:
Does this mean the front end will need to know when a node has that disabled and not show the option to request for access, or will there be an email sent to the institutional admin to let them know that there request couldn't be processed because the option was turned off? Either one has consequences for the timeline, I'd be willing to bet, especially the first one.

Reply:
Regardless of how the FE handles it, I've changed the check to give better info so FE can decide and added tests to demonstrate. I think that's good enough for this PR, Product designs are favoring this approach. Removing the button preemptively likely means changes to SHARE.